flint.ui/src/views/gcbm/GCBMConfigurations.vue

```diff
@@ -1,15 +1,76 @@
-<!-- <template>
-  <div class="flex flex-wrap">
-    <div class="w-full lg:w-8/12 px-4">
-    </div>
-    <div class="w-full lg:w-4/12 px-4">
+<template>
```

Comment:
This file looks like it's just a copy of the RothC template? Suggest we remove and tackle GCBM config later, go straight from upload to run.

Reply:
In a separate PR!

Reply:
Oh it was there just to serve as a filler route, that way we could experiment upon...Sure will update this to the GCBM configs view.

Reply:
Gotcha, just leave an empty placeholder then :😉